scripts/cat/cats.py

```diff
@@ -75,8 +75,10 @@ class Cat():
         "master": (321, 321)
     }
 
+
     default_pronouns = [
         {
+            "name": "Neutral",
```

Comment:
Are default pronouns defined here, or in the JSON resource file? These should only be defined once.